pkgs/applications/editors/kakoune/plugins/case.kak.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit }:
+stdenv.mkDerivation {
+  name = "case.kak";
+  version = "unreleased-2020-04-06";
```

```suggestion
  version = "unstable-2020-04-06";
```

(From [the manual](https://nixos.org/nixpkgs/manual/#sec-package-naming))